frameworks/elastic/universe/config.json

```diff
@@ -315,6 +315,11 @@
           "type": "string",
           "default": ""
         },
+        "custom_yaml_block_base64": {
```

Comment:
What is the future plan for integration with the DC/OS UI?

Reply:
Pretty sure I won't know until _after_ you know. See [this comment](https://jira.mesosphere.com/browse/ELASTIC-108?focusedCommentId=121970&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-121970) in Jira.

Reply:
This same field will be used when the UI is ready. It will handle encoding and decoding from base64. @loren we should probably rename this field. It's supposed to be only for plugin so maybe we should call it 'plugin_config'.  We should ask Keith.  Otherwise good to go.

Reply:
@loren: Please go with `custom_yaml` as the field name.  In the description say, "This field must be base64 encoded...".  Then go ahead and merge once something reasonable has passed in CI. /cc @triclambert

Reply:
@loren:  Actually, just heard back from Keith.  Config value should be named `custom_elasticsearch_yml`.

Reply:
Spoke with @gabrielhartmann about this. We landed on `custom_elasticsearch_yml` in case we need to support other plugins that use files other than elasticsearch.yml.